data_selection/static/data_selection.js

```diff
@@ -131,9 +215,23 @@ function showCurrentRate() {
   $('#audio-speed-rate').text(audioRate + 'x');  
 }
 
+function checkCompletion(){
+  if (input.length > minInputLength) {
+    $('#submit-button').removeClass('disabled');
+  }else {
```

Comment:
Add a space between } and else

Reply:
We should use [prettier](https://prettier.io/) for this... we should not fix this stuff manually